package.json

```diff
@@ -39,13 +40,15 @@
     "semantic-release-slack-bot": "^1.6.2",
     "typescript": "^3.9.7"
   },
-  "dependencies": {
-    "koa2-formidable": "^1.0.2",
+  "peerDependencies": {
```

Comment:
I moved admin-bro, koa, formidable and router to peerDependency - the rule if thumb is to move there everything that you export from your library.